src/components/folder/AddLink.jsx

```diff
@@ -1,30 +1,50 @@
-import { useState } from "react";
 import { styled } from "styled-components";
 import link from "../../assets/link.svg";
+import { useState } from "react";
 import CommonModal from "../modal/CommonModal";
 
-export default function AddLink() {
-  const [addFolderModal, setAddFolderModal] = useState(false);
+export default function AddLink({ folderData }) {
+  const [inputValue, setInputValue] = useState("");
+  const [addLinkModal, setAddLinkModal] = useState(false);
```

Comment:
boolean에 관한 값으로 정할떄는 변수명을 보통 접두사로 is를 붙여주는게 가독성이 좋을때가 많아요!
```suggestion
  const [isAddLinkModal, setIsAddLinkModal] = useState(false);
```